src/pydvl/utils/numeric.py

```diff
@@ -227,3 +249,7 @@ def top_k_value_accuracy(
     top_k_pred_values = np.argsort(y_pred)[-k:]
     top_k_accuracy = len(np.intersect1d(top_k_exact_values, top_k_pred_values)) / k
     return top_k_accuracy
+
+
+def sets_are_equal(subset_1: NDArray[np.int_], subset_2: NDArray[np.int_]):
+    return list(subset_1) == list(subset_2)
```

Comment:
This should be better than a conversion:

```suggestion
    return np.isin(subset_1, subset_2).all()
```

Could you please add a docstring to this function?

Reply:
The conversion to list would fail to implement set equality because the order would matter. Nevertheless I think the function is too trivial to be added. Also, conversion to `set` is probably more intuitive

```python 
set(a) == set(b)
```

So I would just delete this and do the conversion inline wherever necessary

Reply:
@AnesBenmerzoug Removed function and thus docstring not necessary, for equality one would need both inclusions `np.setxor1d(a, b).size == 0`.